models/dataSource.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (sequelize, DataTypes) => {
+  const DataSource = sequelize.define('DataSource', {
+    name: DataTypes.STRING,
+    type: DataTypes.ENUM('InMemory', 'Postgres'),
+    config: DataTypes.JSON
+  })
+
+  return DataSource
+}
```

Comment:
@pb82 fyi, here is the sequelize model for a Data Source, based on the schema being used for the UI
(i.e. https://github.com/aerogear/data-sync-ui/blob/master/gql/schema.js#L4-L16)

Reply:
@david-martin thanks, we will use the same.